        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Contacts:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QListWidget" name="lstContacts">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionScan_Lan"/>
    <addaction name="actionSet_nickname"/>
    <addaction name="actionBalloon_Notification"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear_current_window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionScan_Lan">
   <property name="text">
    <string>Scan LAN</string>
   </property>
  </action>
  <action name="actionSet_nickname">
   <property name="text">
    <string>Set nickname</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBalloon_Notification">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Balloon Notification</string>
   </property>
  </action>
  <action name="actionClear_current_window">
   <property name="text">
    <string>Clear current tab</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSend</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendMsg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>893</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>895</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstContacts</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>newPrivateConvo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendMsg()</slot>
  <slot>newPrivateConvo()</slot>
 </slots>
</ui>
